src/datagen/generated/allomancy/assets/allomancy/lang/en_us.json

```diff
@@ -88,6 +89,8 @@
   "item.allomancy.mistcloak": "Mistcloak",
   "key.cosmere.allomancy.pull": "Pull",
   "key.cosmere.allomancy.push": "Push",
+  "key.cosmere.allomancy.soothe": "Soothe",
```

Comment:
Did you add these manually? The language files in the generated folder gets replaced, so you will need to add them to allomancy eng lang gen

Reply:
I did and didn't know, I'll do that... Once I figure it out